delta/plugins/storage/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/generators/FileGen.scala

```diff
@@ -48,9 +48,11 @@ trait FileGen { self: Generators with FileFixtures =>
 
   def genFileIdWithTag(projRef: ProjectRef): FileId = FileId(genString(), UserTag.unsafe(genString()), projRef)
 
-  def genAttributes(): NonEmptyList[FileAttributes] = {
+  def genAttributesAndMetadata(): NonEmptyList[(FileAttributes, Option[FileUserMetadata])] = {
```

Comment:
I should've done these generators in a less half-arsed way or got scalacheck working 😬 does the job though